app.js

```diff
@@ -577,20 +551,56 @@ app.view('document_help_request', async ({ ack, body, view, client }) => {
     try{
         await ack();
 
-        //console.log(JSON.stringify(body, null, 2));
+        const helpRequestMessages = (await client.conversations.replies({
+            channel: reportChannelId,
+            ts: body.view.private_metadata,
+            limit: 200, // after a thread is 200 long we'll break but good enough for now
+        })).messages
 
-        const documentation = {
-            what: body.view.state.values.what_block.what.value,
-            where: body.view.state.values.where_block.where.value,
-            how: body.view.state.values.how_block.how.value,
-        };
+        {
```

Comment:
```suggestion
```

Reply:
Removed unnecessary block and reformatted.